tests/specs/scoped.js

```diff
@@ -82,6 +82,58 @@ test('scoped Store .find()', function (t) {
   .catch(t.fail)
 })
 
+test('scoped Store .find(array) items not found', function (t) {
+  t.plan(2)
+
+  var store = new Store('test-db-scoped-find', merge({remote: 'test-db-scoped-find'}, options))
+  var testStore = store('test')
+
+  var testItems = [{ foo: 'bar1' }, { foo: 'bar2' }]
+
+  testStore.add(testItems)
+
+  .then(function (items) {
+    return testStore.find(items.map(function (item) { return {id: item.id + '0'} }))
```

Comment:
as we just want to test how `.find()` behaves if none of the times can be found, I’d suggest we do something more simple and explicit, like

```js
return testStore.find([{id: 'non-existing1'}, {id: 'non-existing2'})
```